docs/reference.rst

```diff
@@ -770,6 +770,20 @@ and return a value.
     >>> LogFactory(timestamp=now - timedelta(days=1))
     <Log: log at 2016-02-11 17:02:34>
 
+:class:`LazyFunction` is also useful for assigning copies of mutable objects
+(like lists) to an objects' property. Example:
```

Comment:
typo (assuming you mean `object` singular): `objects'` --> `object's`

Reply:
You're right, thanks.